documentation/CHANGES.rst

```diff
@@ -6,6 +6,36 @@ Overview
 
 The list of the most significant changes made over time in oneDPL.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added a new implementation for ``par`` and ``par_unseq`` execution policies based on OpenMP* 4.5 pragmas.
+  It can be enabled with the ``ONEDPL_USE_OPENMP_BACKEND`` macro.
+  For more details, see `the Macros`_ page in the Library Guide.
+- Added the range-based version of the ``reduce_by_segment`` algorithm and improved performance of
+  the iterator-based ``reduce_by_segment`` APIs. 
+  Please note that the use of the ``reduce_by_segment`` algorithm requires C++17.
+- Added the following algorithms (serial versions) to `the Tested Standard C++ API`_: ``for_each_n``, ``copy``,
+  ``copy_backward``, ``copy_if``, ``copy_n``, ``is_permutation``, ``fill``, ``fill_n``, ``move``, ``move_backward``.
+
+Changes affecting backward compatibility
+----------------------------------------
+- Fixed ``param_type`` API of random number distributions to satisfy C++ standard requirements.
+  The new definitions of ``param_type`` are not compatible with incorrect definitions in previous library versions.
+  Recompilation is recommended for all codes that might use ``param_type``.
+
+Fixed Issues
+------------
+- Fixed hangs and errors when oneDPL is used together with oneMKL in DPC++ programs.
```

Comment:
Since this is the first citation of oneMKL, please use the full name: oneAPI Math Kernel Library (oneMKL)

Reply:
Also, this is the first citation of DPC++ (on its own), please use: Data Parallel C++ (DPC++).

Reply:
Done
There was the first citation of DPC++ in CHANGES before, so there I kept it as-is but applied the rest of mentioned changes